mne/gui/__init__.py

```diff
@@ -239,3 +239,39 @@ def kit2fiff():
     from ._kit2fiff_gui import Kit2FiffFrame
     frame = Kit2FiffFrame()
     return _initialize_gui(frame)
+
+
+@verbose
+def locate_ieeg(info, aligned_ct, subject=None, subjects_dir=None, groups=None,
+                verbose=None):
+    """Locate intracranial electrode contacts.
+
+    Parameters
+    ----------
+    %(info)s
+    aligned_ct : str | pathlib.Path | nibabel.spatialimages.SpatialImage
+        The CT image that has been aligned to the Freesurfer T1. Path-like
+        inputs and nibabel image objects are supported.
+    %(subject)s
+    %(subjects_dir)s
+    groups : dict | None
+        A dictionary with channels as keys and their group index as values.
+        If None, the groups will be inferred by the channel names. Channel
+        names must have a format like ``LAMY 7`` where a string prefix
+        like ``LAMY`` precedes a numeric index like ``7``. If the channels
+        are formatted improperly, group plotting will work incorrectly.
+        Group assignments can be adjusted in the GUI.
+    %(verbose)s
+
+    """
+    from ._ieeg_locate_gui import IntracranialElectrodeLocator
+    from PyQt5.QtWidgets import QApplication
+    import matplotlib as mpl
+    # change to correct backend
+    mpl.use('Qt5Agg')
+    app = QApplication(['Intracranial Electrode Locator'])
```

Comment:
You should not instantiate a QApplication directly, first you check for `.instance` and if it's none you instantiate. Look at how our own `mne/viz/backends/_pyvista.py` and `mne/conftest.py` do it. In this case I think we should refactor the version in `_pyvista.py` to some new `mne.viz._qt` perhaps that just contains code to instantiate a QApplication and set it to use our icon (if we're the ones instantiating it). The `_pyvista.py` code already does this check-and-set, so this code just needs to be moved and used in multiple places

Reply:
Ok I can give that a shot